readme.md

```diff
@@ -46,7 +61,7 @@ Number of columns to wrap the text to.
 
 - [slice-ansi](https://github.com/chalk/slice-ansi) - Slice a string with ANSI escape codes
 - [chalk](https://github.com/chalk/chalk) - Terminal string styling done right
-
+- [jsesc](https://github.com/mathiasbynens/jsesc) - Generate ascii only output from unicode strings. This is useful for creating testing fixtures.
```

Comment:
Why are you not using it programmatically in the tests?

Not really super relevant to this module, but sure, don't mind having it linked.

Description needs some tweaks, though:

> Generate ASCII-only output from Unicode strings. Useful for creating testing fixtures.